tests/jq.test

```diff
@@ -1553,6 +1553,10 @@ bsearch(0,2,4)
 1
```

Comment:
Could you change the query of this case to `bsearch(0,1,2,3,4)`? The output should be `-1 0 1 2 -4`.